tests/constants1.py

```diff
@@ -85,10 +85,10 @@ def test_int():
     a = int(4.56)
     a = int(5)
     a = int(-5.00001)
-    a = int(True)
-    a = int(False)
-    a = int("5346")
-    # a = int(complex(3.45, 5.6)) -> semantic error
+    # commented tests should work
+    # a = int(True)
+    # a = int(False)
+    # a = int("5346")
```

Comment:
For a string I think we need to call a runtime function to do the string to int conversion.